tests/memcheck.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if FLOW_LEAKS=$(grep "definitely lost: [1-9][0-9,]* bytes" flow/logs/*.valgrind.log); then
+	echo "Memory leaks introduced in flow tests:"
+	echo "$FLOW_LEAKS"
+	exit 1
+fi
+
+if TCK_LEAKS=$(grep "definitely lost: [1-9][0-9,]* bytes" tck/logs/*.valgrind.log); then
```

Comment:
Who's responsible for clearing `flow/logs/*.valgrind.log` and `tck/logs/*.valgrind.log` before each run?

Reply:
RLTest is, we specify the argument `--clear-logs` here: https://github.com/RedisGraph/RedisGraph/pull/1002/files#diff-2231ca366d8aa7366f5594abfd41d66aR6